### PR TITLE
Fix statistics page background (fix #2152)

### LINKED ISSUE #2152: background breaks if window is smaller than page
amo-blue-bg-breaks-scroll
the blue background should still stretch the full width of the page.

### PR DESCRIPTION
**Pull request by tofumatt**
### Before

screenshot 2016-04-12 14 27 10
screenshot 2016-04-12 14 29 30

### After

screenshot 2016-04-12 14 27 18
screenshot 2016-04-12 14 28 58

### CODE REVIEW

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -1823,10 +1822,25 @@ h3.author .transfer-ownership {
...
 }
 
-.statistics .primary {
-  float: left;
-  margin-left: 0;
-  width: 77%;
+.restyle.statistics {
+  #page {
+    max-width: @pageWidth;
+    min-width: @pageWidth;
```
Comment (muffinresearch) [inline static/css/restyle/restyle.less]: Couldn't this be `width` or are you overriding?

Comment (tofumatt) [inline static/css/restyle/restyle.less]: Overriding. The old CSS sets a max-width of 1280px on this one page, but it makes it look weird. This brings it in line with the rest f the site.

Comment (muffinresearch) [inline static/css/restyle/restyle.less]: Ah ok gotcha.

**Comment by muffinresearch** [comment]
r+wc

**Pull request merged**